test/metabase/query_processor/middleware/catch_exceptions_test.clj

```diff
@@ -104,37 +104,50 @@
                  :metadata
                  (update :stacktrace boolean)))))))
 
-(deftest include-query-execution-info-test
-  (testing "Should include info from QueryExecution if added to the thrown/raised Exception"
+(deftest catch-exceptions-test
+  (testing "include-query-execution-info-test"
```

Comment:
this is just moving another test named `include-query-execution-info-test` inside of a catchall `catch-exceptions-test` with the former test name being a `testing` context now.